types/src/primary.rs

```diff
@@ -560,23 +558,23 @@ pub enum PrimaryMessage<PublicKey: VerifyingKey> {
 /// Message to reconfigure worker tasks.
 #[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(bound(deserialize = "PublicKey: VerifyingKey"))]
```

Comment:
`#[serde(bound(deserialize = "PublicKey: VerifyingKey"))]` this is not needed any more right?

Reply:
Same comment applies for all the other similar instances